egs2/commonvoice/asr1/conf/tuning/train_asr_transformer.yaml

```diff
@@ -48,4 +48,4 @@ best_model_criterion:
    - max
 keep_nbest_models: 10
 
-init: chainer
+init: xavier_uniform
```

Comment:
Maybe you can leave some comments that this is very important?

Reply:
Sounds good